dist/index.html

```diff
@@ -7,24 +7,26 @@
   <title>Document</title>
 <script defer src="main.js"></script></head>
 <body>
-  <h1>Leaderboard</h1>
-  <div id="main">
-    <section id="section-recent-scores">
-      <div id="head-recent-scores">
+  <main>
+    <h1>Leaderboard</h1>
+    <div id="sub-main">
+      <section id="section-recent-scores">
+        <div id="head-recent-scores">
+          <h3>Recent Scores</h3>
+          <button id="update-button">Refresh</button>
+        </div>
+        <ul id="scores"></ul>
+      </section>
+      <section id="section-add-score">
         <h3>Recent Scores</h3>
```

Comment:
Wouldn't it be better if you follow the heading hierarchy? Consider using <h2> instead of <h3> if the previous one was an <h1> and so on.